fili-wikipedia-example/README.md

```diff
@@ -20,8 +20,16 @@ This example is an entirely self contained example that provides a Fili applicat
     mvn install
     mvn -pl fili-wikipedia-example exec:java
     ```
+    - Note that if your setup is different you can adjust it by changing the default parameters below
+        ```bash
+        # cd fili
+        # mvn install
+        mvn -pl fili-generic-example exec:java -Dbard__fili_port=9998 \
+        -Dbard__druid_coord=http://localhost:8081/druid/coordinator/v1 \
+        -Dbard__druid_broker=http://localhost:8082/druid/v2 \
+        ```
     From another window, run a test query against the default druid data:
-
+    _(Make sure the port matches the `-Dbard__fili_port` argument if you customized it in the previous step)_
```

Comment:
Let's add this line to `fili-generic-example` too

Reply:
changed.